playwright-tests/tests/landing.page.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { expect } from '@playwright/test';
+
+import { test } from '../utils/fixtures';
```

Comment:
Would be better to use full path instead of context path? In case we move files here and there this will break easily.

Reply:
good point! I will make adjustment